packages/Confirmation/src/Confirmation.js

```diff
@@ -82,7 +83,7 @@ const Confirmation = props => {
   }, [confirmButtonRef, defaultIsOpen]);
 
   React.useEffect(() => {
-    popoverKey = uuid();
+    popoverKey = nanoid();
```

Comment:
Just wondering, whats the difference?

Reply:
it's very small and the id is shorter, I noticed we start using it in new components